### DISCUSSION TITLE
How to disable JSON failed logs?

### DISCUSSION THREAD

**Discussion by gotoAndBliss** [cypress-io/cypress#16844] [CI setup]
Cypress 7.4.0

Any failed tests auto generates a file `log/<name of failed spec>.json` .

I have no need for this file. How can I disable this?

**Comment by jennifer-shehane** [comment]
@gotoAndBliss This is not the intended default behavior of Cypress when ran by itself. 

Do you have some outside reporters or plugins setup that may be generating this log?
